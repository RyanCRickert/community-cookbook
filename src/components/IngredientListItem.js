import React from "react";

export default (props) => (
  <div className="ingredient-item">
    <div className="ingredient-item__left">
      {props.name}
    </div>
    <div
      className="ingredient-item__right"
      onClick={(e) => {
        props.handleRemoveIngredient(props.name)
      }}>
      Remove   
    </div>
  </div>
)