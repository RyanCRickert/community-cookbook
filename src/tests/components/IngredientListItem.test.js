import React from "react";
import { shallow } from "enzyme";
import IngredientListItem from "../../components/IngredientListItem";
import ingredients from "../fixtures/ingredients";

test("should render IngredientListItem correctly", () => {
  const wrapper = shallow(<IngredientListItem {...ingredients}/>)
  expect(wrapper).toMatchSnapshot();
});