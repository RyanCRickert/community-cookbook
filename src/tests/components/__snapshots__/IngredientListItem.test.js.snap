// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render IngredientListItem correctly 1`] = `
<div>
  <div
    className="ingredient-item"
  >
    <div
      className="ingredient-item__left"
    />
    <div
      className="ingredient-item__right"
      onClick={[Function]}
    >
      Edit
    </div>
  </div>
</div>
`;
